src/pages/index.tsx

```diff
@@ -1,91 +1,62 @@
+import { GraphQLClient } from "graphql-request";
+import { workoutsQuery, mealsQuery } from "../lib/queries";
 import FitCard from "../components/Card";
 import CardSlider from "../components/CardSlider";
 
-export default function Home() {
-  const workouts = [
-    {
-      id: 0,
-      title: "Killer Pull Workout",
-      image: "assets/images/placeholder.png",
-      description: "Nice Pull Workout",
-      time: 60
-    },
-    {
-      id: 1,
-      title: "Killer Pull Workout",
-      image: "assets/images/placeholder.png",
-      description: "Nice Pull Workout",
-      time: 60
-    },
-    {
-      id: 2,
-      title: "Killer Pull Workout",
-      image: "assets/images/placeholder.png",
-      description: "Nice Pull Workout",
-      time: 60
-    },
-  ]
-
-  const meals = [
-    {
-      id: 0,
-      title: "Big Gains Pasta",
-      image: "assets/images/placeholder.png",
-      description: "Lekkere Italiaanse Pasta",
-      time: 90
-    },
-    {
-      id: 1,
-      title: "Big Gains Pasta",
-      image: "assets/images/placeholder.png",
-      description: "Lekkere Italiaanse Pasta",
-      time: 90
-    },
-    {
-      id: 2,
-      title: "Big Gains Pasta",
-      image: "assets/images/placeholder.png",
-      description: "Lekkere Italiaanse Pasta",
-      time: 90
-    },
-    {
-      id: 3,
-      title: "Big Gains Pasta",
-      image: "assets/images/placeholder.png",
-      description: "Lekkere Italiaanse Pasta",
-      time: 90
-    },
-  ]
+export default function Home({ workouts, meals }: any) {
   return (
     <>
       <CardSlider cardSectionHeading="Featured Workouts" cardSectionLink="/workouts">
-        {workouts.map((workout) => {
+        {workouts.length === 0 && (
```

Comment:
You use && but there is only one conditional your checking?

Reply:
fixed the conditional rendering